app/views/admin/form_templates/_form.html.erb

```diff
@@ -3,8 +3,18 @@
   <%= f.input :title %>
 
   <%= f.simple_fields_for :properties do |p| %>
-    <%= p.input :visible, :label => "Show #{p.object.name}" %>
-    <%= p.input :name, :as => :hidden %>
+    <div id="<%= p.object.name %>" class="panel panel-default template-property">
+      <div class="panel-heading">
+        <div class="panel-title">
+          <%= p.object.name %>
+        </div>
+      </div>
+      <div class="panel-body">
+        <%= p.input :visible, :label => "Show" %>
```

Comment:
You're hurting accessibility here by removing the full object name from the label.  Consider a styled fieldset if you really want to convert to shorter labels.

- http://blog.accessibilityassociates.com/2010/11/23/web-accessibility-of-checkboxes-radio-buttons-the-fieldset/
- http://webaim.org/techniques/forms/controls#checkbox